server/build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
     implementation "org.springdoc:springdoc-openapi-ui:${revOpenapi}"
 
     runtimeOnly "org.glassfish.jaxb:jaxb-runtime:${revJAXB}"
+    runtimeOnly 'com.netflix.conductor:conductor-postgres-persistence:3.13.5'
```

Comment:
This will always include postgres as a dependency. We should only include when db.type is Postgres.

Reply:
+1 to what @manan164 said.  This introduces the postgres dependency with a fixed version.